events/ipfs-201.toml

```
@@ -47,79 +47,85 @@ description = """
 # - use 15m, 20m, 30m, 45m, or 60m session slots
 # - use 15m breaks, one in the morning, one or two in the afternoon
 [[timeslots]]
-startTime="09:00"
-speakers=["@speaker_name", "@other_speaker_name"]
-title="Welcome standup discussion"
-description=""
+startTime="10:00"
+speakers=["Daniel Norman"]
+title="Track introduction"
+description="**Dapps** are all the rave, but there's no consensus about how they should be built. \n\n There's a cambrian explosion of developer tools and many emerging design patterns using IPFS. This introduction will set the stage for some of the fantastic speakers we have during the day who'll be sharing the state of the art in app design patterns and developer tools."
```

```suggestion
description="Dapps are all the rave, but there's no consensus about how they should be built. \n\n There's a cambrian explosion of developer tools and many emerging design patterns using IPFS. This introduction will set the stage for some of the fantastic speakers we have during the day who'll be sharing the state of the art in app design patterns and developer tools."
```

Bold text doesn't render, somehow.